lighthouse-core/gather/gatherers/script-elements.js

```diff
@@ -92,47 +64,28 @@ class ScriptElements extends FRGatherer {
       ],
     });
 
-    for (const script of scripts) {
-      if (mainResource && script.content) script.requestId = mainResource.requestId;
-    }
-
     const scriptRecords = networkRecords
+      .filter(record => record.resourceType === NetworkRequest.TYPES.Script)
       // Ignore records from OOPIFs
-      .filter(record => !record.sessionId)
-      // Only get the content of script requests
-      .filter(record => record.resourceType === NetworkRequest.TYPES.Script);
-
-    // If run on a mobile device, be sensitive to memory limitations and only request one
-    // record at a time.
-    const scriptRecordContents = await runInSeriesOrParallel(
-      scriptRecords,
-      record => fetchResponseBodyFromCache(session, record.requestId).catch(() => ''),
-      formFactor === 'mobile' /* runInSeries */
```

Comment:
I removed the content from this artifact because it is costly, and pubads won't need it.

This is a public artifact, but in my mind.... `content?: string` is OK to go to `content: undefined` :P

Reply:
I mean the performance should only affect pubads then right? Seems like we could play it safe for a breaking rls.

Reply:
The gatherer runs when there is no filters too.

I'd rather remove the content.